README.md

```diff
@@ -9,39 +9,32 @@
 
 ## Installation
 
-    npm install jstransformer-reshape
+`npm install jstransformer-reshape`
 
 > **Note:** Reshape requires node v6 or above.
 
 ## API
 
 ```js
-var reshape = require('jstransformer')(require('jstransformer-reshape'));
+const reshape = require('jstransformer')(require('jstransformer-reshape'))
+const customElements = require('reshape-custom-elements')
 
-var text = `
+const text = `
```

Comment:
I changed these just because you need to be using node v6 anyways for it to work, so why not stick with better es6 conventions 😁